Library/Homebrew/dev-cmd/bump-revision.rb

```diff
@@ -0,0 +1,86 @@
+require "formula"
+require "cli_parser"
+
+module Homebrew
+  module_function
+
+  def bump_revision_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump-revision` [<options>] [<formula>]
+
+        Create a commit to increment the revision of the formula. If no revision is
+         present, "revision 1" will be added.
+      EOS
+      switch "-n", "--dry-run",
+        description: "Print what would be done rather than doing it."
```

Comment:
Is this a useful flag to have? I'm wondering if we need it for a v1 and it makes the code more complex.

Reply:
I think so, it's useful at least for debugging. Combined with the changes from @scpeters it is only two additional `if` block.